### PR TITLE
adding member statefulset missed in 'existing shared PVC' PR

### PR DESCRIPTION
**Pull request by zendril**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Missed the member nodes as part of the previous 'existing shared PVC' PR

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
@eldada @rimusz @jainishshah17 I missed this file in the original PR :(
It is the working, tested version that we are using internally but I just missed it in the commit.
It is also the same pattern that is used on the primary that was committed in the previous PR if you want to compare.

### CODE REVIEW

File: stable/artifactory-ha/templates/artifactory-node-statefulset.yaml [hunk 1]
```diff
@@ -347,7 +380,7 @@ spec:
         secret:
           secretName: {{ tpl . $ }}
       {{- end }}
-      {{- end }}
+      {{- end }}
```
Comment (eldada) [inline stable/artifactory-ha/templates/artifactory-node-statefulset.yaml]: Remove extra spaces

Comment (zendril) [reply stable/artifactory-ha/templates/artifactory-node-statefulset.yaml]: Done

**Comment by zendril** [comment]
@eldada Done.

File: stable/artifactory-ha/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
-## [0.12.6] - Aprl 03, 2019
+## [0.12.7] - Aprl 04, 2019
+* Apply the existing PVC for members (in addition to primary)
+
+## [0.12.7] - Aprl 03, 2019
```
Comment (eldada) [inline stable/artifactory-ha/CHANGELOG.md]: Fix version to `0.12.6`

Comment (zendril) [reply stable/artifactory-ha/CHANGELOG.md]: doh :(

Comment (zendril) [reply stable/artifactory-ha/CHANGELOG.md]: rebased

**Comment by zendril** [comment]
> LGTM
> @zendril - Last rebase before the weekend...

Done. Thanks.

**Comment by zendril** [comment]
@jainishshah17 @eldada Rebase done and Travis seems happy. Good to merge?
Have a great weekend btw. :)

**Comment by zendril** [comment]
Hold off just a few minutes.. Looking at a possible bug in the binarystore template.

**Comment by zendril** [comment]
Ok, good to go. 

It was an extra </config> that showed up, and broke when using non-shared pvc.

**Pull request merged**